packages/sdk/identity/src/odis/bls-blinding-client.ts

```diff
@@ -1,7 +1,7 @@
 import { randomBytes } from 'crypto'
 
 export interface BlsBlindingClient {
-  blindMessage: (base64PhoneNumber: string) => Promise<string>
+  blindMessage: (base64PhoneNumber: string, seed?: Uint8Array) => Promise<string>
```

Comment:
TODO: add documentation specifying that developers should use private and deterministic seed (i.e. the user's dek)

Reply:
TODO: create ticket for Valora to start providing a deterministic private seed